documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: CONCEPT
```

Comment:
I don't think this is necessary if the file name has the `con_` prefix.

Reply:
Removed.